### ISSUE TITLE
Triggering rules and one node in correct and error flow. (0/3)

### ISSUE THREAD

**Issue by mik-laj** [GoogleCloudPlatform/oozie-to-airflow#132]
Depends on #131 

Hello,

The current implementation does not properly support nodes when used in both correct and error flow.
The problem is described in detail in the document
https://github.com/GoogleCloudPlatform/cloud-composer/tree/master/oozie-to-airflow#known-limitations
Visualization of the issues.
<img width="395" alt="Screenshot 2019-05-04 at 13 58 03" src="https://user-images.githubusercontent.com/12058428/57178690-c280e680-6e74-11e9-9549-538c533b2d71.png">

It is possible to bypass this limitation by adding a new task
<img width="477" alt="Screenshot 2019-05-04 at 13 57 58" src="https://user-images.githubusercontent.com/12058428/57178688-beed5f80-6e74-11e9-92b7-adf87bb5f966.png">
